bridge/src/multi-planetary.ts

```diff
@@ -0,0 +1,56 @@
+export interface IPlanetIds {
+    odin: string;
+    heimdall: string;
+}
+
+export interface IPlanetVaultAddress {
+    heimdall: string;
+}
+
+export class MultiPlanetary {
+    private readonly _planetIds: IPlanetIds;
+    private readonly _planetVaultAddresses: IPlanetVaultAddress;
+    constructor(
+        planetIds: IPlanetIds,
+        planetVaultAddresses: IPlanetVaultAddress
+    ) {
+        this._planetIds = planetIds;
+        this._planetVaultAddresses = planetVaultAddresses;
+    }
+
+    isMultiPlanetRequest(_to: string): boolean {
+        const multiPlanetIdRegex = /^[0-9]0{10}[0-9]*$/;
+        return multiPlanetIdRegex.test(_to.substring(2, 14));
+    }
+
+    getRequestPlanetName(_to: string): string {
+        let planetName = "odin";
```

Comment:
define "odin" as MainPlanet or Original Planet as variable and return Main Planet or Original Planet.

Reply:
Fixed to define constant MAIN_PLANET_NAME and set value to "odin"